users/gurtatiLND/website/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+    background-color: rgb(210, 173, 173);
+    box-sizing: border-box;
+    position:relative;
+    line-height: 1.5;
+    /*overflow-x: hidden;
+    overflow-y: scroll;*/
+  }
+
+h1,h2,p {
+   text-align: center;
+  }
+
+/*home*/
+.home {
+    color:rgb(246, 244, 241);
+    background-image: url(./assets/home_background.jpg);
+    background-color:rgb(52, 100, 100);
+    background-position: center;
+    height: 90vh;
+    /*min-height: 80rem;
+    max-height: 120rem;*/
+    position: relative;
+}
+.home_content {
```

Comment:
Follow the kebab-case convention when naming identifiers. Fe 

.home-content {
 ...
}